src/core/MotionPathControls.tsx

```diff
@@ -45,46 +49,49 @@ type MotionState = {
   next: THREE.Vector3
 }
 
-const isObject3DRef = (ref: any): ref is React.MutableRefObject<THREE.Object3D> =>
-  ref?.current instanceof THREE.Object3D
+export type MotionPathRef = THREE.Group & { motion: MotionState }
 
-const context = /* @__PURE__ */ React.createContext<MotionState>(null!)
+const isObject3DRef = (ref: any): ref is React.MutableRefObject<THREE.Object3D> => ref?.current instanceof THREE.Object3D
```

Comment:
please replace `any` with proper type

Reply:
This is okay as a simplification with unknown.